src/ControlFlowGraph.cpp

```diff
@@ -87,6 +87,16 @@ void ControlFlowGraph::newBasicBlock()
     currentBasicBlock = newBasicBlock;
 }
 
+void ControlFlowGraph::setEpilogGenerated()
+{
+    epilogGenerated = true;
+}
+
+bool ControlFlowGraph::getEpilogGenerated() const
```

Comment:
Since epilogGenerated is a boolean calling getEpilogGenerated isEpilogGenerated will make more sense